Documentation/ceph-common-issues.md

```diff
@@ -768,3 +768,15 @@ This problem will be solve by the following two fixes.
 * Ceph: A fix that uses the above-mentioned kernel's feature. The Ceph community will probably discuss this fix after releasing Linux v5.6.
 
 You can bypass this problem by using ext4 or any other filesystems rather than XFS. Filesystem type can be specified with `csi.storage.k8s.io/fstype` in StorageClass resource.
+
+## Too few PGs per OSD warning is shown
```

Comment:
Please add this to the table of contents links at the top of this topic so it is more discoverable

Reply:
Added, thanks.